sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -239,7 +239,9 @@ descriptor against the limitations outlined below. If the given descriptor is
 deemed invalid, then a `sycl::exception` will be thrown with error code 
 `sycl::errc::invalid`.
 
-For all image types, the value of `num_channels` must be `1`, `2`, or `4`.
+The value of `num_channels` supported by all image types and backends
+is  `1`, `2`, or `4`.
+Some backends also support `num_channels` to be `3`.
```

Comment:
How can an application tell whether `3` is supported?

Reply:
Currently there's no way, it just depends on the backend. We want to add this in a later patch, either an aspect or, more likely, a device info descriptor. But that will be more work, this patch is supposed to be the minimal change required to allow 3 channel formats.

Reply:
Can you capture this as an open issue at the bottom of the spec, then?

Reply:
Added a note in the `Issues` section: https://github.com/intel/llvm/pull/15210/commits/f30fbaaae2b5a2ae6b155a9972a038fa030aa181